libs/domains/service-helm/feature/src/lib/networking-setting/networking-setting.tsx

```diff
@@ -29,8 +33,14 @@ export function NetworkingSetting({
   children,
 }: NetworkingSettingProps) {
   const { openModal, closeModal } = useModal()
+  const { openModalMultiConfirmation } = useModalMultiConfirmation()
   const { openModalConfirmation } = useModalConfirmation()
 
+  const { data: customDomains } = useCustomDomains({
+    serviceId: helmId,
+    serviceType: ServiceType.HELM,
```

Comment:
Optional: You can probably use 'HELM' directly it avoid import ServiceType if the type is correct from the hook

```suggestion
    serviceType: 'HELM',
```